core/liquidity/v2/engine.go

```diff
@@ -430,19 +410,18 @@ func (e *Engine) CalculateSuppliedStake() *num.Uint {
 // Dost not include pending commitments.
 func (e *Engine) CalculateSuppliedStakeWithoutPending() *num.Uint {
 	supplied := num.UintZero()
-
 	for _, provision := range e.provisions.ProvisionsPerParty {
 		supplied.AddSum(provision.CommitmentAmount)
 	}
-
 	return supplied
 }
 
 func (e *Engine) IsPoTInitialised() bool {
 	return e.suppliedEngine.IsPoTInitialised()
 }
 
-func (e *Engine) UpdateMarketConfig(model risk.Model, monitor PriceMonitor) {
+func (e *Engine) UpdateMarketConfig(model RiskModel, monitor PriceMonitor, slaParams *types.LiquiditySLAParams) {
+	e.onSLAParamsUpdate(slaParams)
```

Comment:
Why have you decided that the only way to change the SLA params should be through the `UpdateMarketConfig` method?

Reply:
because it's **market** parameters. how else can it be changed?

Reply:
point is the only way to change any of these params that is a market parameter is through a governance update which must include everything (the risk model, price monitor, and sla params, etc.) so there's not really a point in having separate functions to change each one individually.